tools/azure-sdk-tools/packaging_tools/common_utils.py

```diff
@@ -0,0 +1,124 @@
+import glob
+import json
+import logging
+import os
```

Comment:
The file is used by `auto_package` and `auto_codegen`, so the name is better to be `auto_common_utils.py`

Reply:
done